src/Header/Header.tsx

```diff
@@ -73,6 +77,7 @@ const ArrowLeftButton = ({ handleClick }: { handleClick: () => void }) => {
     <button
       className={classNames(styles.headerButton, styles.noBorder)}
       onClick={handleClick}
+      aria-label="arrowLeft"
```

Comment:
Aria labels are announced by screen readers. Make it human friendly
```suggestion
      aria-label="Previous"
```